output/cloud/output.go

```diff
@@ -328,7 +328,7 @@ func (out *Output) getRunStatus(testErr error) cloudapi.RunStatus {
 }
 
 func (out *Output) startVersionedOutput() error {
-	if out.referenceID == "" {
+	if out.testRunID == "" {
 		return errors.New("ReferenceID is required")
```

Comment:
Maybe also adjust an error to be more precise?

```suggestion
		return errors.New("RefetenceID (Test Run ID) is required")
```